plugins/field-slider/test/field_slider_test_helpers.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const assert = require('chai').assert;
+const FieldSlider = require('../dist/index').FieldSlider;
+
+// Monica's note
```

Comment:
Suggestion: add your notes behind a TODO, that way eslint will remind you they're there. eg: 
```
// TODO (Monica): ... 
```

Reply:
Thanks. I'm removing a lot of them for now, but I'll probably use the TODO for any future comments while I'm working.